pkg/api/universal/scheduler.go

```diff
@@ -21,12 +21,13 @@ import (
 	apierrors "github.com/dapr/dapr/pkg/api/errors"
 	runtimev1pb "github.com/dapr/dapr/pkg/proto/runtime/v1"
 	schedulerv1pb "github.com/dapr/dapr/pkg/proto/scheduler/v1"
+	"github.com/dapr/dapr/pkg/security"
 )
 
 func (a *Universal) ScheduleJob(ctx context.Context, inReq *runtimev1pb.ScheduleJobRequest) (*emptypb.Empty, error) {
 	metadata := map[string]string{
-		"appId":     a.AppID(),
-		"namespace": a.globalConfig.Namespace,
+		"appID":     a.AppID(),
```

Comment:
Im pretty sure we discussed standardizing on `appID`. Additionally, the `a.globalConfig.Namespace` is unset. Ik that `security.CurrentNamespace()` works and contains the proper ns. This is important for the streaming logic to work, as it needs the ns && appID passed along in the metadata for the connPool stream lookup later

Reply:
I am using `security.CurrentNamespace()` instead of `security.CurrentNamespaceOrError()` since it was erroring for me due to me running in standalone mode, but not setting the `NAMESPACE` env var. I think we should not error out and kill daprd if a user is running in standalone mode and but not setting the env var.

Reply:
Local DevEx does not set NAMESPACE and it is not required, unless dev is using mTLS.

Reply:
Also "appID" is Go style, while JSON and Yaml is "appId". At this point, I saw that we are using "appID" for errors - which is not the standard for JSON. This is minor, so let's just pick one and have it as a constant we can reference vs a hardcoded string.

Reply:
`appID` it is :)